EIPS/eip-225.md

```diff
@@ -0,0 +1,474 @@
+---
+eip: 225
+title: Clique proof-of-authority consensus protocol
+author: Péter Szilágyi <peter@ethereum.org>
+discussions-to: https://github.com/ethereum/EIPs/issues/225
+status: Final
+type: Standards Track
+category: Consensus
```

Comment:
I would say that a consensus sensitive EIP should be classified as:

```
Type: Standard Track
Category: Core
```

Reply:
I don't really agree, because it's not a change to the consensus rules, it's a completely new network. You can't hard for to upgrade to it. We don't have such a category that would cater for these kinds of EIPs, but I think we would get quite a few if we allowed (tendermint, istanbul, aura, clique, etc).

Reply:
I would like to see this merged to go into further iterations from here. I temp to agree with Nick that we should not make the process more complicated than it already is and submit EIP-225 as Core EIP. If we discuss Core vs Consensus, then we have to go down the rabbit hole thinking about if this an EIP.

My suggestion: Mark this as Core, and open another PR to suggest a Consensus track in EIP-1. So we can have both discussions contained.

Reply:
We're trying to separate EIPs from what gets deployed on the main network. Under that system, "Core" doesn't mean "on mainnet", it means "relating to consensus rules", so I think core is just fine here.

Reply:
> My suggestion: Mark this as Core, and open another PR to suggest a Consensus track in EIP-1. So we can have both discussions contained.

I would agree this is the best past in order to get this merged.

@karalabe do you have any objections to this?

Reply:
I think that's a good idea.